extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.js

```diff
@@ -35,15 +35,37 @@ angular.module('PaperUI.controllers', [ 'PaperUI.constants' ]).controller('BodyC
         return uuid;
     };
 
-    var numberOfInboxEntries = -1;
+    var numberOfInboxEntries = -1, prevAudioUrl = '';
+    window.AudioContext = window.AudioContext || window.webkitAudioContext;
+    if (typeof (window.AudioContext) != "undefined") {
+        var context = new AudioContext();
+    }
     eventService.onEvent('smarthome/inbox/*/added', function(topic, discoveryResult) {
         toastService.showDefaultToast('New Inbox Entry: ' + discoveryResult.label, 'Show Inbox', 'inbox/search');
     });
     eventService.onEvent('smarthome/webaudio/playurl', function(topic, audioUrl) {
-        if (audioUrl) {
-            var audio = new Audio(audioUrl);
-            audio.load();
-            audio.play();
+        if (prevAudioUrl !== audioUrl) {
+            if (context) {
+                var audioBuffer = null;
+                var arr = audioUrl.split('/');
```

Comment:
Please note that with https://github.com/eclipse/smarthome/pull/2374, the audioUrl is now only a relative url starting with a "/", e.g. "/audio/12345". So I assume you can slightly optimize this code here.